### DISCUSSION TITLE
请教下R66S下载passwall版本的问题？

### DISCUSSION THREAD

**Discussion by bobvane** [istoreos/istoreos#1472] [Q&A]
装passwall在iStoreOS里面，在下载页面那个luci-19.07和luci-23.05有什么区别，而R66S的iStoreOS又应该下载19.07还是23.05呢？
![屏幕截图 2024-04-03 135850](https://github.com/istoreos/istoreos/assets/58875973/07864b20-5b6e-44eb-8e19-93bd6c3aad6b)

**Comment by jjm2473** [comment]
应该都一样，我不觉得有人会重写passwall，估计也就依赖不一样。
